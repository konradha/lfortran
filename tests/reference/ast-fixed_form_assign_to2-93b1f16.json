{
    "basename": "ast-fixed_form_assign_to2-93b1f16",
    "cmd": "lfortran --indent --fixed-form --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form_assign_to2.f",
    "infile_hash": "ac85cc11719c45a4dc882205861d93a7375f2ce2a85289f800d2170e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-fixed_form_assign_to2-93b1f16.stdout",
    "stdout_hash": "ebe50287c5262ce861ac3bbbd176c560ef2b9a36596700041a64f8e5",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}