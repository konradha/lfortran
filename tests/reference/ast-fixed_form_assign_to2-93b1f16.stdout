(TranslationUnit 
    [(Program 
        main 
        () 
        [] 
        [] 
        [] 
        [(If 
            100 
            () 
            (< 0 1) 
            [(Assign 
                0 
                100 
                k 
                ()
            )] 
            [] 
            () 
            () 
            ()
        )
        (GoTo 
            0 
            () 
            (+ k 1) 
            [100
            101] 
            ()
        )
        (Print 
            101 
            () 
            [(String 
                "yes"
            )] 
            ()
        )] 
        []
    )]
)
